saleor/graphql/order/mutations/fulfillments.py

```diff
@@ -44,6 +48,38 @@ class FulfillmentCancelInput(graphene.InputObjectType):
     restock = graphene.Boolean(description="Whether item lines are restocked.")
 
 
+class FulfillmentClearMeta(ClearMetaBaseMutation):
+    class Meta:
+        description = "Clears meta for fulfillment"
+        model = models.Fulfillment
+        permission = "orders.manage_orders"
+        public = True
+
+
+class FulfillmentUpdateMeta(UpdateMetaBaseMutation):
+    class Meta:
+        description = "Updates meta for fulfillment"
+        model = models.Fulfillment
+        permission = "orders.manage_orders"
```

Comment:
We need to confirm that basic meta update/clear should be protected by staff's permissions.